src/hapticfeedback.h

```diff
@@ -0,0 +1,57 @@
+/* antimicrox Gamepad to KB+M event mapper
+ * Copyright (C) 2022 Max Maisel <max.maisel@posteo.de>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include <QString>
+#include <SDL2/SDL_gamecontroller.h>
+#include <stdint.h>
+
+#include "hapticfeedbackmode.h"
+
+class HapticFeedback
+{
+  public:
+    explicit HapticFeedback(HapticFeedbackMode mode = HAPTIC_FEEDBACK_NONE, int strength = 0, int start = 0, int end = 0);
+
```

Comment:
Why HapticFeedback is not `QObject`?

Reply:
It is recommended, because than to this you can for example properly manage memory in your app [example](https://stackoverflow.com/a/2491811/11747535), QT [docs](https://doc.qt.io/qt-5/objecttrees.html)

Reply:
Fixed in next patchset.